processes/sfrCluster.py

```diff
@@ -42,9 +42,10 @@ def runProcess(self):
     def clusterRecords(self, full=False, startDateTime=None):
         baseQuery = self.session.query(Record)\
             .filter(Record.frbr_status == 'complete')\
-            .filter(Record.cluster_status == False)\
+            .filter(Record.cluster_status == True)\
             .filter(Record.source != 'oclcClassify')\
-            .filter(Record.source != 'oclcCatalog')
+            .filter(Record.source != 'oclcCatalog')\
+            .filter(Record.title == 'Democratic Education')
```

Comment:
Does changing these filters potentially affect other script runs?

Reply:
No it doesn't affect them since this is just a temporary change that won't be merged into the main branch. Also, this  ClusterProcess is usually the last process to be run in our daily ingestion process.